doc/releases/release-notes-1.14.rst

```diff
@@ -313,6 +313,14 @@ Drivers and Sensors
   * Added SSD1673 EPD controller driver
   * Added SSD1306 display controller driver
 
+* Device Tree:
+
+  * Added proper support for boolean properties
```

Comment:
Aren't you duplicating some of the info in the "Build and Infrastructure" section?

```
Introduced DT_ prefix for all labels generated for information extracted from device tree (with a few exceptions, such as labels for LEDs and buttons, kept for backward compatibility with existing applications).
Introduce CMake variables for DT symbols, just as we have for CONFIG symbols.
Move DeviceTree processing before Kconfig. Thereby allowing software to be configured based on DeviceTree information.
```

Reply:
Ok merged my changes into the build/infrastructure section.

Reply:
thanks!